vllm_spyre/v1/worker/spyre_worker.py

```diff
@@ -444,7 +444,8 @@ def _warmup_spyre_dynamic_size(self, special_token_ids):
             0, len(valid_token_ids_tensor), (3, prompt_len))]
 
         # TODO: we need 2 requests for warmup on FP8+CB
-        is_fp8_plus_cb = 'FP8' in self.model_config.model and \
+        # Check if model is quantized
```

Comment:
What about other quantizations like 4 bit or 8 bit `int`?

There is some more code from @joerunde and some I recently wrote to do this kind of check, which could be adapted and used here:

- https://github.com/vllm-project/vllm-spyre/blob/de0e3d2c6e18ad21baca4833d947e1f9f5df82f8/vllm_spyre/platform.py#L597
- https://github.com/vllm-project/vllm-spyre/blob/de0e3d2c6e18ad21baca4833d947e1f9f5df82f8/vllm_spyre/config/runtime_config_validator.py#L139

Reply:
Created a follow-up issue: https://github.com/vllm-project/vllm-spyre/issues/537